swipeable-rv/src/main/java/io/huannguyen/swipeablerv/SWItemRemovalListener.java

```diff
@@ -35,6 +39,8 @@
      *         The item being added back
      * @param position
      *         The position of the item being added back
+     * @param direction
```

Comment:
You added a new `direction param` while the method definition is not updated. I think there is no benefit of passing `direction` when an item is added back to the list, so probably no change should be made here. What do you think?